Source/OxyPlot/PlotModel/PlotModel.Rendering.cs

```diff
@@ -500,7 +500,8 @@ private void UpdatePlotArea(IRenderContext rc)
 
             // Find the available size for the legend box
             var availableLegendWidth = plotArea.Width;
```

Comment:
why is availableLegendWidth not consistent with availableLegendHeight? can this line be made consistent with the other?